setup.py

```diff
@@ -26,7 +26,7 @@ def read(*parts):
             'django-cadmin = configurations.management:execute_from_command_line',
         ],
     },
-    install_requires=[],
+    install_requires=['django'],
```

Comment:
Should this be
```suggestion
    install_requires=['django>=2.2'],
```
as that's what `django-configurations` supports?